script.js

```diff
@@ -0,0 +1,573 @@
+      sessionStorage.clear();
+      sessionStorage.setItem('object', JSON.stringify(objectKeysRU));// add object in session storage
+      objectKeysRU = JSON.parse(sessionStorage.getItem('object'));
+    } else {
+      for (let i = 1; i < 65; i += 1) {
+        [...KEYBOARD.querySelectorAll('div')][i - 1].innerHTML = objectKeys[i];
+        [...KEYBOARD.querySelectorAll('div')][i - 1].id = objectId[i];
+      }
+      if (CAPS.classList.contains('active-caps')) CAPS.classList.remove('active-caps');
+      sessionStorage.clear();
+      sessionStorage.setItem('object', JSON.stringify(objectKeys)); // add object in session storage
+      objectKeys = JSON.parse(sessionStorage.getItem('object'));
+    }
+  }
+};
+
+const RU = document.getElementById('ru');
+RU.addEventListener('click', () => {
+  if ([...KEYBOARD.querySelectorAll('div')][60].innerHTML === 'RU') {
+    for (let i = 1; i < 65; i += 1) {
+      [...KEYBOARD.querySelectorAll('div')][i - 1].innerHTML = objectKeysRU[i];
+      [...KEYBOARD.querySelectorAll('div')][i - 1].id = objectIdRU[i];
+    }
+    sessionStorage.clear();
+    sessionStorage.setItem('object', JSON.stringify(objectKeysRU));// add object in session storage
+    objectKeysRU = JSON.parse(sessionStorage.getItem('object'));
+  } else {
+    for (let i = 1; i < 65; i += 1) {
+      [...KEYBOARD.querySelectorAll('div')][i - 1].innerHTML = objectKeys[i];
+      [...KEYBOARD.querySelectorAll('div')][i - 1].id = objectId[i];
+    }
+    sessionStorage.clear();
+    sessionStorage.setItem('object', JSON.stringify(objectKeys));// add object in session storage
+    objectKeys = JSON.parse(sessionStorage.getItem('object'));
+  }
+});
+
+// Language and register definition
+document.addEventListener('keydown', (event) => {
+  if (event.key.length === 1) {
+    if (/[a-z]{1,1}/.test(event.key)) {
+      for (let i = 1; i < 65; i += 1) {
+        [...KEYBOARD.querySelectorAll('div')][i - 1].innerHTML = objectKeys[i];
+        [...KEYBOARD.querySelectorAll('div')][i - 1].id = objectId[i];
+      }
+    } else if (/[A-Z]{1,1}/.test(event.key)) {
+      for (let i = 1; i < 65; i += 1) {
+        [...KEYBOARD.querySelectorAll('div')][i - 1].innerHTML = objectKeys[i].toUpperCase();
+        [...KEYBOARD.querySelectorAll('div')][i - 1].id = objectId[i];
+      }
+    } else if (/[А-Я]{1,1}/.test(event.key)) {
+      for (let i = 1; i < 65; i += 1) {
+        [...KEYBOARD.querySelectorAll('div')][i - 1].innerHTML = objectKeysRU[i].toUpperCase();
+        [...KEYBOARD.querySelectorAll('div')][i - 1].id = objectIdRU[i];
+      }
+    } else if (/[а-я]{1,1}/.test(event.key)) {
+      for (let i = 1; i < 65; i += 1) {
+        [...KEYBOARD.querySelectorAll('div')][i - 1].innerHTML = objectKeysRU[i];
+        [...KEYBOARD.querySelectorAll('div')][i - 1].id = objectIdRU[i];
+      }
+    }
+  }
+  textArea.focus();
+});
+
+document.addEventListener('keypress', (event) => {
+  const s = event.key;
+  if (event.key.toUpperCase() === s) {
```

Comment:
I faced a problem, when typed a capital letter using Left Shift. 
![Screen Shot 2020-04-08 at 8 46 11 PM](https://user-images.githubusercontent.com/26601879/78816299-35c29f00-79da-11ea-9c87-f4387994bff9.png)

Reply:
fixed
thank you